packages/typespec-ts/src/index.ts

```diff
@@ -266,7 +264,13 @@ export async function $onEmit(context: EmitContext) {
     const isMultiClients = dpgContext.sdkPackage.clients.length > 1;
 
     emitTypes(dpgContext, { sourceRoot: modularSourcesRoot });
-    buildSubpathIndexFile(dpgContext, modularEmitterOptions, "models");
+    buildSubpathIndexFile(
```

Comment:
One thing to confirm - we don't have a flag to turn on/off with model's namespace adoption because all generation would adopt that, right?

Reply:
Just notice you mentioned that.